nutkit/frontend/driver.py

```diff
@@ -67,6 +67,28 @@ def send_and_receive(self, req, timeout=None, hooks=None, *,
         return self.receive(timeout=timeout, hooks=hooks,
                             allow_resolution=allow_resolution)
 
+    def execute_query(self, cypher, params=None, config=None,
```

Comment:
```suggestion
    def execute_query(self, cypher, params=None,
```
unused parameter, I believe.